bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -312,6 +313,16 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
     return (clientTransactionSigningEnabled || !clientsKeysPrefix.empty()) ? &publicKeysOfClients : nullptr;
   }
 
+  std::string getOperatorPublicKey() {
```

Comment:
How will we distinguish between the situation when it's not an operator and a one when the should be a key, but it wasn't provided?

Reply:
We don't do it at this point. The responsibility of ReplicaConfig is only to parse the operator key if it exists, not to assert if it wasn't provided and needed to be.
So, it is some higher level responsibility to assert this scenario. I mean, if it happens, then it should affect many more components, this problem is not a problem specifically related to this change.